internal/scheduler/proposer_duties.go

```diff
@@ -0,0 +1,48 @@
+package scheduler
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-co-op/gocron/v2"
+	"github.com/shutter-network/gnosh-metrics/common/utils"
+	"github.com/shutter-network/gnosh-metrics/internal/metrics"
+)
+
+const propserDutiesSchedulerDuration = "* * * * *" //every minute
```

Comment:
querying the proposer duty every minute means every 12 slots. One epoch is 16 slots. I would prefer to query it in a fraction of an epoch. Let's say every 1/2 epoch (which would be 8 slots -> 40 seconds).